src/modules/trading-view/services/trading.view.service.ts

```diff
@@ -0,0 +1,200 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import {
+    BarsQueryArgs,
+    BarsResponse,
+    TradingViewResolution,
+    resolutionMapping,
+} from '../dtos/bars.response';
+import { AnalyticsQueryService } from 'src/services/analytics/services/analytics.query.service';
+import { PriceCandlesResolutions } from 'src/modules/analytics/models/query.args';
+import moment from 'moment';
+import BigNumber from 'bignumber.js';
+import { decodeTime } from 'src/utils/analytics.utils';
+import { TokenService } from 'src/modules/tokens/services/token.service';
+import { RouterAbiService } from 'src/modules/router/services/router.abi.service';
+import { EsdtToken } from 'src/modules/tokens/models/esdtToken.model';
+import { PairMetadata } from 'src/modules/router/models/pair.metadata.model';
+
+@Injectable()
+export class TradingViewService {
+    constructor(
+        private readonly analyticsQueryService: AnalyticsQueryService,
+        private readonly tokenService: TokenService,
+        private readonly routerAbi: RouterAbiService,
+    ) {}
+
+    async resolveSymbol(
+        symbol: string,
+    ): Promise<{ ticker: string; name: string }> {
```

Comment:
use a type for return

Reply:
Added in https://github.com/multiversx/mx-exchange-service/pull/1386